app/src/main/java/com/xmartlabs/template/ui/onboarding/signUp/SignUpViewModel.kt

```diff
@@ -0,0 +1,27 @@
+package com.xmartlabs.template.ui.onboarding.signUp
+
+import androidx.annotation.MainThread
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.Transformations
+import androidx.lifecycle.ViewModel
+import com.xmartlabs.template.model.User
+import com.xmartlabs.template.usecase.SignUpUseCase
+import javax.inject.Inject
+
+class SignUpViewModel @Inject constructor() : ViewModel() {
+  private val _signUp = MutableLiveData<SignUpUseCase.Params>()
```

Comment:
i think here is better to name this like `signUpMutableLiveData` or something like that instead of `_`

Reply:
the _ is a pattern that google is using, (just a comment, you can change it)